pilot/pkg/serviceregistry/kube/conversion.go

```diff
@@ -123,7 +123,7 @@ func serviceHostname(name, namespace, domainSuffix string) model.Hostname {
 
 // canonicalToIstioServiceAccount converts a Canonical service account to an Istio service account
 func canonicalToIstioServiceAccount(saname string) string {
-	return fmt.Sprintf("%v://%v", IstioURIPrefix, saname)
+	return fmt.Sprintf("%v://accounts.google.com/%v", IstioURIPrefix, saname)
```

Comment:
%s ?

Reply:
Per the discussion, this PR keeps the original %v.